service/history/timerQueueProcessor.go

```diff
@@ -104,3 +146,85 @@ func (t *timerQueueProcessorImpl) getTimerFiredCount(clusterName string) uint64
 	}
 	return standbyTimerProcessor.getTimerFiredCount()
 }
+
+func (t *timerQueueProcessorImpl) completeTimersLoop() {
```

Comment:
Let's create a separate task to add emit some metric from this new logic.  I think we are missing key metric which would help us investigate issues in production.

Reply:
sure

Reply:
https://github.com/uber/cadence/issues/680